README.md

```diff
@@ -96,4 +96,113 @@ You can customize the authorization flow by subclassing view classes in [`eidas_
 Specific Connector
 ------------------
 
-Not yet implemented.
+Sample settings are provided in [`samples/connector_settings.py`](samples/connector_settings.py).
+
+### Usage
+
+To use eIDAS Connector, adjust Django settings:
+
+* Set up  Django template engine and static files.
+* Add `'eidas_node.connector.apps.ConnectorConfig'` to `INSTALLED_APPS`.
+* Set `ROOT_URLCONF` to `'eidas_node.connector.urls'` or include it in your URL configuration.
+* Provide mandatory configuration options `CONNECTOR_REQUEST_TOKEN`, `CONNECTOR_RESPONSE_TOKEN`, `CONNECTOR_LIGHT_STORAGE`, and `CONNECTOR_SERVICE_PROVIDER` (see below).
+
+### Views
+
+URL configuration at `eidas_node.connector.urls` provides following views:
+
+* `/CountrySelector`:
+  Accepts a *SAML Request* and *Relay State* from Service Provider and lets a user select its *country* unless it has already been provided.
```

Comment:
Probably `... and lets user select his/her ...` looks little a bit better?

Reply:
I wanted to use a gender-neutral pronoun, but it's `their` not `its`. `his/her` is also ok.